samples/todo-android/app/src/androidTest/java/com/squareup/sample/mainactivity/TodoAppTest.kt

```diff
@@ -23,19 +25,40 @@ class TodoAppTest {
   @Rule @JvmField val scenarioRule = ActivityScenarioRule(ToDoActivity::class.java)
   private val uiDevice by lazy { UiDevice.getInstance(getInstrumentation()) }
 
-  @Test fun navigatesToListAndBack_portrait() {
-    uiDevice.setOrientationNatural()
+  @Before
+  fun setUp() {
+    uiDevice.unfreezeRotation()
```

Comment:
Are these both supposed to be unfreeze?

Reply:
Yes, defensive in case some other test screws up the hellaciously sticky VM. `setOrientationLeft` and like calls freeze the orientation.